.mergify.yml

```diff
@@ -1,5 +1,17 @@
+queue_rules:
+  - name: default
+    conditions:
+      - status-success=build (compile-linux)
+      - status-success=build (compile-windows)
+      - status-success=build (compile-darwin)
+      - status-success=test
+      - status-success=integ-test
+      - status-success=staticcheck
+      - status-success=license
+      - status-success=Semantic Pull Request
+
```

Comment:
Why do we need this 🤔

Reply:
I didn't dig deep into it, but AFAICT now PRs:
1. First get queued (`conditions` under `pull_request_rules`)
2. Then they get merged if the conditions in the queue are satisfied (this new section)

So for example, if there are 2 approvals and no "WIP" label and CI is passed, then the PR gets queued to be merged, then Mergify checks again if the CI statuses are all valid and if so merges it.